src/js/formers/op-build-helpers.js

```diff
@@ -1,15 +1,16 @@
-import { DocumentContainer } from '@/js/helpers/DocumentContainer'
+import { base } from '@tokend/js-sdk'
+import { DOCUMENT_POLICIES } from '@/js/const/document-policies.const'
 
 export function doc (doc) {
```

Comment:
now that we have uploadDocumentsDeep, and `toJSON` implementation on Document, this stuff looks redundant, as the only thing we need is to call `uploadDocumentsDeep(form.documents)`

Reply:
this one for me, I’ll update this accordingly after the PR is merged